amundsen_application/api/preview/dashboard/v0.py

```diff
@@ -0,0 +1,51 @@
+import io
+import logging
+from http import HTTPStatus
+
+from flask import send_file, jsonify, make_response, Response, current_app as app
+from flask.blueprints import Blueprint
+
+from amundsen_application.dashboard_preview.preview_factory_method import DefaultPreviewMethodFactory, \
+    BasePreviewMethodFactory
+
+LOGGER = logging.getLogger(__name__)
+PREVIEW_FACTORY: BasePreviewMethodFactory = None  # type: ignore
+
+dashboard_preview_blueprint = Blueprint('dashboard_preview', __name__, url_prefix='/api/dashboard_preview/v0')
+
+
+def initialize_preview_factory_class() -> None:
+    """
+    Instantiates Preview factory class and assign it to PREVIEW_FACTORY
+    :return: None
+    """
+    global PREVIEW_FACTORY
```

Comment:
Note: We plan on switching to restful endpoints in the future since the blueprints are stateless and that approach made us have to do things like this to not recreate the client each time. https://github.com/lyft/amundsen/issues/307. Christina also started using them for the JIRA issues [here](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/api/issue/issue.py). 

No need to change at this point though! I could have communicated that before you started working on this.

Reply:
> Note: We plan on switching to restful endpoints in the future since the blueprints are stateless and that approach made us have to do things like this to not recreate the client each time. [lyft/amundsen#307](https://github.com/lyft/amundsen/issues/307). Christina also started using them for the JIRA issues [here](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/api/issue/issue.py).
> 
> No need to change at this point though! I could have communicated that before you started working on this.

Sounds good. May be we can do separate task to do the migration?

Reply:
Yeah separate task for sure. I'm fine with this as is, but for future work in this repo you can use restful endpoints and I will eventually get rid of blueprints.